src/sentry/replays/serializers.py

```diff
@@ -10,3 +12,67 @@ def serialize(self, obj: ReplayRecordingSegment, attrs, user):
             "projectId": str(obj.project_id),
             "dateAdded": obj.date_added,
         }
+
+
+VALID_FIELD_SET = {
+    "id",
+    "title",
+    "projectId",
+    "errorIds",
+    "traceIds",
+    "urls",
+    "startedAt",
+    "finishedAt",
+    "duration",
+    "countErrors",
+    "countSegments",
+    "countUrls",
+    "longestTransaction",
+    "platform",
+    "environment",
+    "release",
+    "dist",
+    "user",
+    "os",
+    "browser",
+    "device",
+    "sdk",
+    "tags",
+}
+
+
+class ReplaySerializer(serializers.Serializer):
+    statsPeriod = serializers.CharField(
+        help_text=(
+            "This defines the range of the time series, relative to now. "
+            "The range is given in a `<number><unit>` format. "
+            "For example `1d` for a one day range. Possible units are `m` for minutes, `h` for hours, `d` for days and `w` for weeks."
+            "You must either provide a `statsPeriod`, or a `start` and `end`."
+        ),
+        required=False,
+    )
+    start = serializers.DateTimeField(
+        help_text="This defines the start of the time series range as an explicit datetime, either in UTC ISO8601 or epoch seconds."
+        "Use along with `end` instead of `statsPeriod`.",
+        required=False,
+    )
+    end = serializers.DateTimeField(
+        help_text=(
+            "This defines the inclusive end of the time series range as an explicit datetime, either in UTC ISO8601 or epoch seconds."
+            "Use along with `start` instead of `statsPeriod`."
+        ),
+        required=False,
+    )
+    field = serializers.ChoiceField(
+        VALID_FIELD_SET,
+        help_text="Specifies a field that should be marshaled in the output. Invalid fields will be rejected.",
+        required=False,
+    )
+    project = serializers.ListField(
```

Comment:
should we specify a child of `serializers.IntegerField()` here?

Reply:
@JoshFerge Some use list fields.  Some user integer fields.  Some user char fields.  I think list is the most accurate because our api accepts multiple project parameters.

Reply:
right, but you can specify what type should go in the list